components/automate-ui/src/app/pages/integrations/list/integrations-list.component.spec.ts

```diff
@@ -54,4 +59,27 @@ describe('IntegrationsListComponent', () => {
   it('exists', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('deleting a node manager', () => {
+    it('displays a modal prompt when user selects Delete Integration from the dropdown', () => {
+      component.beginDelete(MockSelectionEvent, '12345');
```

Comment:
Better to first establish a baseline, confirming the prompt was not visible. (Like you did in the next test. 😁 )